### DISCUSSION TITLE
Temperature Room

### DISCUSSION THREAD

**Discussion by feanor-anglin** [sindrebroch/ha-flexit#22] [Q&A]
Hi, I have a question about Temperature Room sensor. Do I need any additional hardware to see the value of this sensor? Right now I have it at 0 degrees.

Is this a different value than Temperature Extract? Why can't this two sensors be combined?

**Comment by sindrebroch** [comment]
Hi. Thanks for your question.

Weird, for me it reports a different temperature than temperature extract, but not by much. Not sure if I can see it in Flexit Go.

Which Flexit unit do you have?

**Reply by feanor-anglin** [reply]
Thank you for lightning fast answer. The model is POS3.67.

I've tried to replace the sensor in climate entity (because it now shows a huge 0 degrees), but unsuccessfully.

**Reply by sindrebroch** [reply]
Hmm, yeah the room temperature sensor is used for the climate-entity. Did not know this was missing for some people. I'll look into what would be a good way of handling this. 

It should not create any issues in the functionality itself, only visual bugs for the entity in HomeAssistant.

**Reply by feanor-anglin** [reply]
I suggest replacing it in climate entity with extract temperature in case it is equal to 0.

**Reply by sindrebroch** [reply]
Yes will probably do something like that.

**Reply by FSHelgeland** [reply]
I also miss this as well, I have the CL4 Nordic.  I have the exctract temp and the supply temp, but room temp is stating 0.0

**Comment by DTekNOdev** [comment]
What was the fix for this? I have 0 degrees for the "room sensor", and also 0 for current sensor with the climate entity, The model is given as POS3.6715/414. Flexit Nordic CL3.
